llvm/test/Transforms/LoopVectorize/iv-select-cmp-decreasing.ll

```diff
@@ -965,25 +965,141 @@ exit:                                             ; preds = %loop
 ; In this test, %iv's range will include both signed and unsigned
 ; maximum (sentinel) values.
 define i64 @not_vectorized_select_decreasing_induction_icmp_iv_out_of_bound(ptr %a, ptr %b, i64 %rdx.start) {
-; CHECK-LABEL: define i64 @not_vectorized_select_decreasing_induction_icmp_iv_out_of_bound(
```

Comment:
This test description/name needs updating as it's currently intended to be a negative test.